Added content matching score FAQ (#24)
ansible/ansible-lightspeed: docs/faq/index.md

The permissions `Read org and team membership, read org projects` allow Ansible Lightspeed to verify participants in the closed beta program; those users belong to a specific team that is verified upon authorization.  Ansible Lightspeed does not use the permission to read other organizational repository data or use GitHub API data for any other purpose.

## What is the content matching score and how is it calculated?

> The score returned from a k-nearest neighbor (kNN) search that is generated by a similarity function (which determines the distance between two vectors). The score is always greater than zero and larger scores imply a "shorter distance", or greater similarity, between two strings.

Reviewer: ```suggestion
The score returned from a k-nearest neighbor (kNN) search that is generated by a similarity function (which determines the distance between two vectors) and then converted into a search score; larger scores imply a greater similarity between two strings.
```
